### ISSUE TITLE
Adding no logit option

### ISSUE THREAD

**Issue by sidml** [Hsuxu/Loss_ToolBox-PyTorch#3]
This is an excellent implementation of focal loss. 
I had a small suggestion. Perhaps, you may want to give the user an option to use logit directly.

Currently the code assumes that the input is non-logit.
`logit = F.softmax(input, dim=1)`

**Comment by Hsuxu** [comment]
Your suggest is very constructive and will be considered.
